_docs/sdk/android.md

```diff
@@ -258,6 +258,18 @@ Smartlook.trackCustomEvent("duration_event")
 ```
 In this case `duration_event` will have duration property set to circa `1000ms`.
 
+#### Custom name for clicked events
+
+Sdk detects click events automatically. Name of the event constists of `Activity` name, `View` type (`Button`, etc.) and `id`.
```

Comment:
Use capital letters = `SDK`

constists -> consists

Name of the event -> The event name consists of ...